runelite-client/src/main/java/net/runelite/client/plugins/randomevents/RandomEventPlugin.java

```diff
@@ -55,28 +55,30 @@
 public class RandomEventPlugin extends Plugin
 {
 	private static final Set<Integer> EVENT_NPCS = ImmutableSet.of(
-		NpcID.DR_JEKYLL, NpcID.DR_JEKYLL_314,
 		NpcID.BEE_KEEPER_6747,
 		NpcID.CAPT_ARNAV,
 		NpcID.SERGEANT_DAMIEN_6743,
+		NpcID.DUNCE_6749,
 		NpcID.DRUNKEN_DWARF,
+		NpcID.EVIL_BOB, NpcID.EVIL_BOB_6754,
+		NpcID.FLIPPA_6744,
 		NpcID.FREAKY_FORESTER_6748,
+		NpcID.FROG_5429,
 		NpcID.GENIE, NpcID.GENIE_327,
-		NpcID.EVIL_BOB, NpcID.EVIL_BOB_6754,
-		NpcID.POSTIE_PETE_6738,
+		NpcID.DR_JEKYLL, NpcID.DR_JEKYLL_314,
 		NpcID.LEO_6746,
 		NpcID.MYSTERIOUS_OLD_MAN_6750, NpcID.MYSTERIOUS_OLD_MAN_6751,
 		NpcID.MYSTERIOUS_OLD_MAN_6752, NpcID.MYSTERIOUS_OLD_MAN_6753,
+
+		NpcID.GILES, NpcID.GILES_5441,
+		NpcID.MILES, NpcID.MILES_5440,
+		NpcID.NILES, NpcID.NILES_5439,
+
 		NpcID.PILLORY_GUARD,
-		NpcID.FLIPPA_6744,
+		NpcID.POSTIE_PETE_6738,
 		NpcID.QUIZ_MASTER_6755,
 		NpcID.RICK_TURPENTINE, NpcID.RICK_TURPENTINE_376,
-		NpcID.SANDWICH_LADY,
-		NpcID.DUNCE_6749,
-		NpcID.NILES, NpcID.NILES_5439,
-		NpcID.MILES, NpcID.MILES_5440,
-		NpcID.GILES, NpcID.GILES_5441,
-		NpcID.FROG_5429
+		NpcID.SANDWICH_LADY
```

Comment:
Instead of the pseudo-alphabetizing happening here, can this simply be fully alphabetized? (eg. ***S***ergeant Damien, ***D***r Jekyll, separate Giles/Miles/Niles, etc.)

Reply:
done